Samples/VirtualCamera/VirtualCameraManager_WinRT/VirtualCameraRegistrar.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once
+#include "VirtualCameraRegistrar.g.h"
+#include "winrt/Windows.Devices.Enumeration.h"
+
+namespace winrt::VirtualCameraManager_WinRT::implementation
+{
+    struct VirtualCameraRegistrar : VirtualCameraRegistrarT<VirtualCameraRegistrar>
+    {
+        VirtualCameraRegistrar() = delete;
+
+        static winrt::VirtualCameraManager_WinRT::VirtualCameraProxy RegisterNewVirtualCamera(
+            winrt::VirtualCameraManager_WinRT::VirtualCameraKind const& virtualCameraKind, 
+            winrt::VirtualCameraManager_WinRT::VirtualCameraLifetime const& lifetime, 
+            winrt::VirtualCameraManager_WinRT::VirtualCameraAccess const& access, 
+            hstring const& friendlyName,
+            hstring const& realCameraSymbolicLink);
```

Comment:
>realCameraSymbolicLink [](http://example.com/codeflow?start=27&length=22)

suggested rename - phyisicalCameraSymbolicLink?

Reply:
it is not necessarily as physical camera

Reply:
existing maybe?

Reply:
parentCamera?

Reply:
wrappedCamera